src/pages/agreement/Agreement.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useRef } from "react";
+import Select from "react-select";
+import { Form, Button } from "react-bootstrap";
+import ShowAgreement from "./ShowAgreement";
+const { data } = require("./exampleData");
+
+const universityList = [
+  { label: "UCLA", value: "UCLA" },
+  { label: "UC Berkeley", value: "UC Berkeley" },
+  { label: "UCSD", value: "UCSD" },
+];
+
+const ccList = [
+  { label: "DeAnza College", value: "DeAnza College" },
+  { label: "Foothill College", value: "Foothill College" },
+];
+
+//this can be done by ajax
+const majorList = [
+  { label: "Physics/B.A.", value: "Physics/B.A." },
+  { label: "Mathematics/B.S.", value: "Mathematics/B.S." },
+  { label: "Data Theory/B.S.", value: "Data Theory/B.S." },
+  { label: "Computer Science/B.S.", value: "Computer Science/B.S." },
+  {
+    label: "Asian American Studies/B.A.",
+    value: "Asian American Studies/B.A.",
+  },
+  { label: "Anthropology/B.S.", value: "Anthropology/B.S." },
+  { label: "Labor Studies/B.A.", value: "Labor Studies/B.A." },
+  { label: "Global Studies/B.A.", value: "Global Studies/B.A." },
+  { label: "Psychobiology/B.S.", value: "Psychobiology/B.S." },
+];
```

Comment:
It would be better if the frontend and backend can share the same university/cc/major list.

Reply:
Yes, this part will be store in our db in the future, and will be requested from db when user select the target university and cc. The dropdown menu for majors will be render dynamiclu based on that.

Reply:
Shall we open a new repo called AssistFetch to fetch uni/cc/major list from assist.org?

Reply:
This sounds great.